seed_reader/lib/routes/generate_seed/generate_seed_page.dart

```diff
@@ -0,0 +1,171 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import '../../../generated/l10n.dart';
+import '../../components/countdown_timer.dart';
+import '../../design_tokens/dimensions.dart';
+import '../../errors.dart';
+import '../../models/seed.dart';
+import '../../routes/generate_seed/generate_seed_page_bloc.dart';
+
+class GenerateSeedPage extends StatelessWidget {
+  const GenerateSeedPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(S.of(context).generateSeedPageTitle),
+      ),
+      body: Consumer<GenerateSeedPageBloc>(
+        builder: (_, GenerateSeedPageBloc bloc, __) {
```

Comment:
chore: you can live off the last argument since we're ignoring it. The line would become `builder: (_, GenerateSeedPageBloc bloc) {`

Reply:
The signature of the builder parameter uses a nullable but not optional parameter, as you can check [here](https://github.com/rrousselGit/provider/blob/master/lib/src/consumer.dart#L175).

It would be nice if the provider changes to an optional parameter.